packages/skeleton/src/lib/components/Autocomplete/Autocomplete.svelte

```diff
@@ -8,22 +8,25 @@
 	type TransitionIn = Transition;
 	type TransitionOut = Transition;
 	type Value = unknown;
+	type Meta = unknown;
 </script>
 
 <script
 	lang="ts"
-	generics="Value = unknown, TransitionIn extends Transition = SlideTransition, TransitionOut extends Transition = SlideTransition"
+	generics="Value = unknown, Meta = unknown, 
+	TransitionIn extends Transition = SlideTransition, TransitionOut extends Transition = SlideTransition"
 >
 	import { createEventDispatcher } from 'svelte';
 	// import { flip } from 'svelte/animate';
 	// import {slide} from 'svelte/transition';
 
 	// Types
 	import type { AutocompleteOption } from './types.js';
+	type Option = AutocompleteOption<Value, Meta>;
```

Comment:
Would love a suggestion if there's a better name for this, haha. 
Since this is reused so much throughout the component I've extracted it to a separate type.

Reply:
Given that the value it represents is named `options`, I think this is perfectly fine 😄